### PR TITLE
auth/multi_chain_sigtx_middleware_test.go : Improve comments

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

File: auth/multi_chain_sigtx_middleware_test.go [hunk 1]
```diff
@@ -263,7 +263,7 @@ func TestBinanceAddressMappingVerification(t *testing.T) {
 		func(state loomchain.State) (contractpb.StaticContext, error) { return amCtx, nil },
 	)
 
-	// Normal loom transaction without address mapping
+	// Normal local transaction without address mapping
```
Comment (enlight) [inline auth/multi_chain_sigtx_middleware_test.go:266]: Should leave this as `loom`

File: auth/multi_chain_sigtx_middleware_test.go [hunk 2]
```diff
@@ -340,32 +340,32 @@ func TestChainIdVerification(t *testing.T) {
...
```
Comment (enlight) [inline auth/multi_chain_sigtx_middleware_test.go:343]: Ditto

File: auth/multi_chain_sigtx_middleware_test.go [hunk 3]
```diff
@@ -194,30 +194,30 @@ func TestEthAddressMappingVerification(t *testing.T) {
...
```
Comment (enlight) [inline auth/multi_chain_sigtx_middleware_test.go:197]: Should leave this one as `loom`

**Comment by andreipope** [comment]
@enlight It's updated

**Pull request merged**